### ISSUE TITLE
ssh regexps incomplete

### ISSUE THREAD

**Issue by thewiwi** [fail2ban/fail2ban#1944]
sshd.conf:  
```INI
mdre-extra = ^Received <F-MLFFORGET>disconnect</F-MLFFORGET> from <HOST>%(__on_port_opt)s:\s*14: No supported authentication methods available%(__suff)s$
            ^Unable to negotiate with <HOST>%(__on_port_opt)s: no matching (?:cipher|key exchange method) found.
            ^Unable to negotiate a (?:cipher|key exchange method)%(__suff)s$
```
but ssh has more related messages that are currently not caught, for example:
```
Oct 26 15:30:40 localhost sshd[14737]: Unable to negotiate with 1.2.3.4 port 50404: no matching MAC found. Their offer: hmac-sha1,hmac-sha1-96,hmac-md5,hmac-md5-96,hmac-ripemd160,hmac-ripemd160@openssh.com [preauth]
```
from ssh 7.6 ssherr.c:
ssherr.c:               return "no matching cipher found";
ssherr.c:               return "no matching MAC found";
ssherr.c:               return "no matching compression method found";
ssherr.c:               return "no matching key exchange method found";
ssherr.c:               return "no matching host key type found";

**Comment by thewiwi** [comment]
same issue:
https://github.com/fail2ban/fail2ban/issues/1943

**Comment by Fuchus** [comment]
That should do the trick:
`^%(__prefix_line_sl)sUnable to negotiate with <HOST>%(__on_port_opt)s:\sno matching (?:cipher|key exchange method|MAC|compression method|host key type) found\.\sTheir offer:.*%(__suff)s$`

**Comment by sebres** [comment]
Fixed in 159957a for v.0.10 and higher. 
For multiline (buffered) variant usable in v.0.9, take a look the changes in example filter [zzz-sshd-obsolete-multiline.conf](https://github.com/fail2ban/fail2ban/commit/159957ab883efc151a2dddf53197153ba41576c2#diff-26ca86257581861d63befe5d65835816).
